src/appComponents/common/ApiJsonFactory.js

```diff
@@ -3,43 +3,42 @@
  */
 (function () {
 
-"use strict";
-const commonFactories = angular.module("openevent");
+  "use strict";
+  const commonFactories = angular.module("openevent");
 
 commonFactories.factory("ApiJsonFactory", ["$q", "$http", function ($q, $http) {
 
-var baseUrl = (config.use_testApi ? "testapi/" : config.apiBaseGetUrl) + "event/" + config.eventId;
+const baseUrl = (config.use_testApi ? "testapi/" : config.apiBaseGetUrl) + "event/" + config.eventId;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'use_testApi' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2445396365/issues/source?bid=3233857&fileBranchId=3245777#l11)